datadog/resource_datadog_synthetics_test_.go

```diff
@@ -253,6 +253,14 @@ func syntheticsTestOptions() *schema.Schema {
 					errs = append(errs, fmt.Errorf("%q.allow_insecure must be either true or false, got: %s", key, allowInsecureStr))
 				}
 			}
+			// retryRaw, ok := val.(map[string]interface{})["retry"]
+			// if ok {
+			// 	retryInt := int
+			// }
+			// if v < 0 || v > 3 {
+			// 	errs = append(errs, fmt.Errorf("%q must be between 0 and 3 inclusive, got: %d", key, v))
+			// }
+			// return
```

Comment:
Probably should remove that? Do you need validation on the number of retries.

Reply:
Yes I removed it, It might be annoying to keep this in sync with the api if it changes